downloadfile.py

```diff
@@ -0,0 +1,10 @@
+import os.path
+import requests
+url='https://samples.adsbexchange.com/readsb-hist/2022/05/01/235745Z.json.gz'
+
+def download_url(url):
+    filename= os.path.basename(url)
+    r = requests.get(url, allow_redirects=True)
+    open('data/' + filename, 'wb').write(r.content)
+
+download_url(url)
```

Comment:
this is going to run every time you import the module somewhere (which you don't want it to do). remove this.

Reply:
you could add a `__main__` and have this inside of the main with the `url` variable from above. this would then only run when you ran `python downloadfile.py`

Reply:
I added an if __name__ == "__main__" but still confused on what should be the desired output.